lasagne/updates.py

```diff
@@ -440,14 +440,17 @@ def rmsprop(loss_or_grads, params, learning_rate=1.0, rho=0.9, epsilon=1e-6):
            Neural Networks for Machine Learning, Lecture 6.5 - rmsprop.
            Coursera. http://www.youtube.com/watch?v=O3sxAc4hxZU (formula @5:20)
     """
+    # Using theano constant to prevent upcasting of float32
+    one = T.constant(1)
+
```

Comment:
Nitpick, but I'd prefer to move this *after* `grads = ` and `updates = ` so all update functions start the same.